ConeSearch.tex

```diff
@@ -1,92 +1,67 @@
+\previousversion[http://www.ivoa.net/Documents/PR/DAL/ConeSearch-20060908.html]{PR 1.00 2006-09-08}
+       
+
+\begin{document}
+
+\begin{abstract} This specification defines a simple
+query protocol for retrieving records from a catalog of astronomical
+sources. The query describes sky position and an angular distance,
+defining a cone on the sky. The response returns a list of astronomical
+sources from the catalog whose positions lie within the cone, formatted
+as a VOTable. This version of the specification is essentially a
+transcription of the original Cone Search specification in order to move
+it into the IVOA standardization process. \end{abstract}
+
+
+\section*{Acknowledgments} 
+
+This document was originally published as a
+document of the US National Virtual Observatory (NVO)\footnote{The NVO
+document: "Conesearch definition" (Roy Williams, Bob Hanish, Alex
+Szalay; April 2002) is currently available at this link:
+\url{https://wiki.ivoa.net/internal/IVOA/ConeSearch/NVO\_conesearch\_April2002.html.}}
+and then transcripted into an IVOA standards document format to become
+an IVOA Recommendation. The changes made in this transcription and in
+the process of producing the first recommendation are reported in
+Appendix \ref{appendix:nvochanges}.
 
 The Simple Cone Search version 1.03 document has been developed with
 support from the National Science Foundation's Information Technology
 Research Program under Cooperative Agreement AST0122449 with The Johns
 Hopkins University.
 
-The ConeSearch-1.1 revision has been developed under the ASTERICS
-project, supported by the European Commission Framework Programme
-Horizon 2020 Research and Innovation action, grant agreement n. 653477.
-This work has been supported by the ESCAPE project (the European Science
-Cluster of Astronomy \& Particle Physics ESFRI Research Infrastructures)
-that has received funding from the European Union’s Horizon 2020
-research and innovation programme under the Grant Agreement n. 824064
-
-Work of the original authors of the Cone Search specification as well as
-the numerous data providers who have implemented and continue to support
-this protocol is kindly acknowledged.
-
-\section*{Conformance-related definitions}
-
-The words ``MUST'', ``SHALL'', ``SHOULD'', ``MAY'', ``RECOMMENDED'', and
-``OPTIONAL'' (in upper or lower case) used in this document are to be
-interpreted as described in IETF standard RFC2119 \citep{std:RFC2119}.
-
-The \emph{Virtual Observatory (VO)} is a general term for a collection
-of federated resources that can be used to conduct astronomical
-research, education, and outreach.  The
-\href{http://www.ivoa.net}{International Virtual Observatory Alliance
-(IVOA)} is a global collaboration of separately funded projects to
-develop standards and infrastructure that enable VO applications.
-
+The ConeSearch-1.1 revision has been developed and supported under the
+ASTERICS and ESCAPE projects (funded by the European Commission
```

Comment:
Is this new version of the ConeSearch, now being re-written differently, still
been developed and supported under the ASTERICS and ESCAPE projects?
  
I would rather say it has been started or initiated by these projects.
Anyway, no strong opinion here ; I was just wondering...
  
  In any case, the corresponding ChangeLog could/should stay in this document.

Reply:
Good point, check whether [c08324d](https://github.com/ivoa-std/ConeSearch/pull/55/commits/c08324da7c4de476b88c5f6a7c734e153bcf177f) works.